sleap/io/video.py

```diff
@@ -1253,6 +1253,8 @@ def from_filename(cls, filename: str, *args, **kwargs) -> "Video":
             kwargs["dataset"] = ""  # prevent serialization from breaking
         elif os.path.isdir(filename) or "metadata.yaml" in filename:
             backend_class = ImgStoreVideo
+        elif filename.lower().endswith(("jpg", "jpeg", "png", "tif", "tiff")):
+            backend_class = SingleImageVideo
```

Comment:
We can add a small easy test in [test/io/test_video.py](https://github.com/talmolab/sleap/blob/develop/tests/io/test_video.py) for `Video.from_filename` using the jpegs in our [test data](https://github.com/talmolab/sleap/tree/develop/tests/data/videos) (i.e. tests\data\videos\robot0.jpg)